app/managers.py

```diff
@@ -0,0 +1,57 @@
+import sqlite3
+import os
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect(
+            os.path.join(os.path.dirname(__file__), "identifier.sqlite")
+        )
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"""
+            INSERT INTO {self.table_name} (first_name, last_name)
+            VALUES (?, ?)
+            """,
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def all(self) -> None:
+        actor_manager_cursor = self._connection.execute(
+            f"""
+            SELECT * FROM {self.table_name}

```

Comment:
should be 
SELECT *
FROM {self.table_name}
SQL keywords must start from new lines